### ISSUE TITLE
Check accuracy when using bigram instead of unigram

### ISSUE THREAD

**Issue by farukcankaya** [TUM-IDP-WS-20/nlp-examples#13]
Base: https://github.com/TUM-IDP-WS-20/nlp-examples/blob/master/8_Milestone_1_Accuracy_Comparison.ipynb
We use '[ngram_range](https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.CountVectorizer.html)' attribute of CountVectorizer with default value that is (1,1) = unigram. Test with (1,2), and (2,2)
